### PR TITLE
[i18n] Update Occitan

### PR DESCRIPTION
**Pull request by Quenty31**

### CODE REVIEW

**Comment by fallen** [comment]
Thanks @Quenty31 for the PR!
Can you describe a bit what this PR changes in Occitan translation?

the file diff is a bit messy since lots of changes are in fact just lines moving around...
It's very hard to follow what exactly has changed.

**Comment by Quenty31** [comment]
Hello!
Somehow weblate reorganised the file, but it went from ~80% translated to 100%.
That's the main information :)

**Comment by fallen** [comment]
Alright, let's merge then!
Thanks a lot again :)
